CppCoreGuidelines.md

```diff
@@ -16867,6 +16867,47 @@ Static helps dynamic: Use static polymorphism to implement dynamically polymorph
 Dynamic helps static: Offer a generic, comfortable, statically bound interface, but internally dispatch dynamically, so you offer a uniform object layout.
 Examples include type erasure as with `std::shared_ptr`'s deleter (but [don't overuse type erasure](#Rt-erasure)).
 
+    #include <memory>
+
+    class Object {
+    public:
+        template <typename T>
```

Comment:
I think this line has a more space between `template <typename T>` compared to other examples